rpm/osg-pki-tools.spec

```diff
@@ -43,6 +43,9 @@ mv %{buildroot}/%{_prefix}/config/ca-issuer.conf %{buildroot}%{_sysconfdir}/osg/
 %config(noreplace) %{_sysconfdir}/osg/pki/ca-issuer.conf
 
 %changelog
+* Wed Sep 6 2023 Brian Lin <blin@cs.wisc.edu> - 3.6.1
+- Fix bug with default CA config file option (SOFTWARE-5668)
+
```

Comment:
Suggest inserting also the updates of igtfservercert and igtfmultidomain done in #80

Reply:
Good call. Fixed